dev_tools/nyaml2nxdl/nyaml2nxdl_helper.py

```diff
@@ -270,7 +270,6 @@ def separate_hash_yaml_and_nxdl(yaml_file, sep_yaml, sep_xml):
             sep_xml, "w", encoding="utf-8"
         ) as xml_f_ob:
             write_on_yaml = True
-
```

Comment:
I would remove this change to not introduce any diff with this PR.

Reply:
It was unintended.